client/app/license/order/license-order.html

```diff
@@ -217,12 +215,13 @@ <h2 data-translate="license_order_step4_title"></h2>
                                     data-ng-if="agoraEnabled"
                                     data-translate="license_order_step4_contracts_agora_ask">
                             </strong>
-                            <span data-ng-repeat="contract in durations.details[selected.duration].contracts">
+                            <span data-ng-repeat="contract in durations.details[selected.duration].contracts track by $id">
                                 <a target="_blank"
                                    rel="noopener"
-                                   class="d-inline-block"
-                                   data-ng-href="{{ contract.url }}"
-                                   data-ng-bind="contract.url">
+                                   class="d-inline-block oui-link_icon"
+                                   aria-label="{{ ::'common_newtab' | translate }}">
```

Comment:
No, this isn't what `aria-label` does : `aria-label` replaces the text of an element by its value. So, a screen reader will think the text of the link is "Opens a new tab".

A better way to do it would be:
```js
<a>
    <span data-translate="common_newtab" class="sr-only"></span>
</a>
```

This will hide the span but screen readers will still read it :)

Reply:
Have added the span. Now i get it...thanks